web/src/components/FilingsCalendar.tsx

```diff
@@ -96,6 +107,20 @@ export const FilingsCalendar = (props: Props): ReactElement => {
             </div>
           );
         })}
+        {sortedFilteredOperatingFilings.length > listViewMoreIncrement && (
+          <div
+            className={"view-more-button"}
+            onClick={() => setNumberOfVisibleCalendarEntries((previous) => previous + listViewMoreIncrement)}
+            onKeyDown={() =>
+              setNumberOfVisibleCalendarEntries((previous) => previous + listViewMoreIncrement)
+            }
+            role="button"
+            tabIndex={0}
+            aria-label={`View ${listViewMoreIncrement} More Calendar Entries`}
+          >
+            View More
+          </div>
```

Comment:
You don't need to recreate all this from scratch (Same with your `view-more-button` css). For one, for semantic html we want to use <button> components instead of div role=button. 

But we keep all the button types in our `Button` component. You can achieve the styling you're looking for with `<Button style="tertiary">